pkg/bootstrap/docker/up.go

```diff
@@ -84,11 +85,11 @@ var (
 	  %[1]s --host-data-dir=/mydata --use-existing-config
 
 	  # Use a different set of images
-	  %[1]s --image="registry.example.com/origin" --version="v1.1"`)
+	  %[1]s --image="registry.example.com/origin" --version="v1.1"
+
+	  # Use a different set of image streams
+	  %[1]s --image-streams="name:relative/path/to/image-streams.json"`)
```

Comment:
relative path to what?  i'm a little confused by what the default behavior is since i thought cluster up embedded the imagestreams it needs/uses by default.  is this actually filesystem path, or is this a path into the map of embedded json data?